loginPresenter

LL333111 opened the pull request.

LL333111 committed:
- loginPresenter is complete

ethanliu24 commented on src/main/java/controller/login/LoginPresenter.java:
```diff
@@ -1,19 +1,42 @@
+        loginState.setEmailError(error);
+        loginViewModel.firePropertyChanged();
+    }
 }
```
Add a new method switchToHomePage and implement it. fix any errors it brings up

ethanliu24 replied:
The new method should be overriden from LoginOutputBoundary

LL333111 committed:
- switchToHomepage is added
- switchToSignupView is added again. I think I implemented this before but I don't know why it disappear so I just add it again.

ethanliu24 reviewed (approved):
lgtm!
